### DISCUSSION TITLE
How to remove node from just one way, without affecting other elements using the node?

### DISCUSSION THREAD

**Discussion by NeatNit** [openstreetmap/iD#10230] [Q&A] [answered]
### Description

I recently mapped something in between two administrative regions, and one of the nodes I created was accidentally added to the border between the regions: https://www.openstreetmap.org/node/11495793691
I didn't notice until later. FWIW this happened in Vespucci, not iD.

Now I want to remove that node from the administrative regions, but not affect any of the other ways. I haven't found a clean way to do this, besides making a mess and then undoing the mess.

Current:
![image](https://github.com/openstreetmap/iD/assets/2400026/8d1a527b-8b0d-4a82-9a5d-c9bf1d37bec1)

Desired:
![image](https://github.com/openstreetmap/iD/assets/2400026/5a92d690-54a1-4c0e-8f63-c6f0d2df8fd7)

Unwanted middle step:
![image](https://github.com/openstreetmap/iD/assets/2400026/47eaf7dc-a4c9-41b3-a59c-d335b882f85b)

I'm not sure it would have even been possible if not for the playground area which happened to be below the administrative boundaries - if it wasn't there to serve as an anchor to re-attach everything to, I don't think it would have been possible without some slight movement of everything. And honestly, it doesn't even matter if some things moved very slightly - it's the principle of the matter, and I want to learn how to use iD better. Ideally I want to make this change with *only* the boundaries in the list of changes; as it stands, the changes shown for submitting show 5 elements changed - the two boundaries, two walking paths and the playground, even though they weren't actually changed.

Note that I haven't submitted the change, so you can have a go. Trying to recreate the scenario in some empty area of the map didn't give me the exact same behavior, with regards to what node gets selected after splitting the node.

**Comment by Dimitar5555** [comment, marked as answer]
1) Select the node and the boundary way using shift and press D
2) Add a new node on the right of the node you want to remove
3) Select the node you've just created and press Page up or Page down. You should be on the node you want to delete. In case you aren't, try this step again with the other Page button.
4) Press Delete or Backspace to delete the node
5) Delete the node you've created in step 2 

P.S. This is an issue tracker for iD. For such questions you can use the OSM community forum (https://community.openstreetmap.org).

**Comment by NeatNit** [comment]
> 1. Select the node and the boundary way using shift and press D
> 
>     2. Add a new node on the right of the node you want to remove
> 
>     3. Select the node you've just created and press Page up or Page down. You should be on the node you want to delete. In case you aren't, try this step again with the other Page button.
> 
>     4. Press Delete or Backspace to delete the node
> 
>     5. Delete the node you've created in step 2

Thanks! Just to confirm, it wouldn't be possible without the Page Up/Down keys, right? I did find that right-click can disconnect the line from the node, but then found no way to select and delete the new node. Shouldn't there be some way to select between overlapping objects that are in the same location?

> P.S. This is an issue tracker for iD. For such questions you can use the OSM community forum (https://community.openstreetmap.org).

Sorry, my bad.. But you better change the New Issue template then:
![image](https://github.com/openstreetmap/iD/assets/2400026/e8538df2-9e94-4e12-a696-dfa6c13f153b)
